bigframes/constants.py

```diff
@@ -84,8 +84,17 @@
         "europe-west9",
         "europe-west8",
         "me-central2",
+        "us-central1",
```

Comment:
Sorry I missed reviewing this earlier, we need to also update the API documentation https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes._config.bigquery_options.BigQueryOptions#bigframes__config_bigquery_options_BigQueryOptions_use_regional_endpoints

Reply:
Thanks for the reminder. Let me use another PR to fix it.